From fd8ab3d597d444bba8d49550dde28929c41f352e Mon Sep 17 00:00:00 2001
From: Robert Mader <robert.mader@collabora.com>
Date: Wed, 28 Jan 2026 15:15:30 +0100
Subject: [PATCH 5/6] aperture: Fix bitrate for openh264enc

The elements uses bits/sec, not kbits/sec like most other encoders.

(cherry picked from commit ed731ee86d891e406d7dc4a1391e65dbf1462ce9)
---
 aperture/src/viewfinder.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aperture/src/viewfinder.rs b/aperture/src/viewfinder.rs
index 9fd6f02..39cf1c7 100644
--- a/aperture/src/viewfinder.rs
+++ b/aperture/src/viewfinder.rs
@@ -927,7 +927,7 @@ impl Viewfinder {
             )
             .item(
                 ElementPropertiesMapItem::builder("openh264enc")
-                    .field("bitrate", DEFAULT_BITRATE)
+                    .field("bitrate", DEFAULT_BITRATE * 1024)
                     .build(),
             )
             .item(
-- 
2.52.0

